### PR TITLE
Adding support for eval

### PR DESCRIPTION
**Pull request by adi-miller**
Adding support to customize the MQTT message payload by using eval().

### CODE REVIEW

File: MMM-MQTT.js [hunk 1]
```diff
@@ -115,12 +120,12 @@ Module.register("MMM-MQTT", {
     getDom: function () {
         self = this;
         var wrapper = document.createElement("table");
-        wrapper.className = "small";
```
Comment (adi-miller) [inline MMM-MQTT.js]: Not related to this change. Will remove.

**Comment by ottopaulsen** [comment]
Thank you for contributing.

I am afraid I don't want to merge code using eval this way. It is too risky a function.  See for example [MDN doc for eval()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval) where it is recommended never to use it, and especially in a place where the text it is run with comes from an external source, like it very well might be in this module.

I can however recommend for example [Node-RED](https://nodered.org/) to handle this kind of requirements.

**Comment by adi-miller** [comment]
OK, although security-wise, I dont think configuration is considered an external source in this case. If someone has access to the config file, they probably have access to the code as well. 

What do you suggest for the purpose of supporting conversion of seconds to minutes in that case? 

Thanks.

**Comment by ottopaulsen** [comment]
It is not only config. Your eval was also evaluating the value coming from the message, and message queues may be far more open than your config.

Node-RED, as I mentioned, is perfect for changing message content, but of course, it requires a bit more setup.

Maybe a "divisor" and/or "multiplier" config variable could be a simple alternative. It could be applied before the rounding is done.

**Pull request closed without merging**